client/js/components/user/orgaRegisterForm/OrgaRegisterForm.vue

```diff
@@ -52,31 +52,37 @@
             <legend :class="prefixClass('font-size-medium is-label u-mb-0_25')">
               {{ Translator.trans('organisation.type') }}
             </legend>
-            <div :class="hasPermission('feature_identity_broker_login') ? prefixClass('space-stack-xs') : prefixClass('o-form__group')">
+            <p :class="prefixClass('u-mb')">
+              {{ Translator.trans('organisation.kind.explanation') }}
+            </p>
+            <div :class="hasPermission('feature_identity_broker_login') ? prefixClass('space-stack-xs') : prefixClass('o-form__group flex-col')">
```

Comment:
I think when removing all classes from the checkbox elements, you could use `space-stack-s` here.

```suggestion
            <div :class="prefixClass('space-stack-s')">
```

Reply:
hmm, it doesn't have any influence when I use the class `space-stack-s`. But you are right, if I remove those classes, it doesn't break the layout

Reply:
`space-stack-s` is overruled by the `u-mb-0_75` on the child elements.

Reply:
hhm, no.. it doesn't have any impact even when I remove the margin. It seems like the class is not being recognized

Reply:
ok, got it. I've removed margins from the checkboxes and added the `space-stack-s` class instead of all these classes